includes/GameDashboard.hpp

```diff
@@ -79,6 +79,35 @@ class GameDashboard
      */
     void decreaseWatchingTime();
 
+    /**
+     * @brief getter of the found star(s) amount
+     *
+     * @return const unsigned short&
+     */
+    const unsigned short& getFoundStarsAmount();
+
+    /**
+     * @brief updates the SFML surface that displays the total amount of cells
+     * on the level; this method is called directly after the end of the level
+     * loading process;
+     *
+     * @param amount the amount to display in the dashboard
+     */
+    void updateStarsAmountSurface(const unsigned short& amount);
```

Comment:
everywhere in the variable names, the difference must be explicit between the "found" and "total" stars amounts